src/tests/ftest/util/daos_utils.py

```diff
@@ -410,6 +418,27 @@ def container_destroy_snap(self, pool, cont, snap_name=None, epoch=None,
             CommandFailure: if the daos pool query command fails.
 
         """
+        if len(epochs) == 1:
+            return self._get_result(
+                ("container", "destroy-snap"), pool=pool, svc=svc, cont=cont,
+                sys_name=sys_name, snap=snap_name, epc=epochs[0])
+
+        epcrange = "{}-{}".format(epochs[0], epochs[1])
```

Comment:
```suggestion
        kwargs = {
            "pool": pool,
            "svc": svc,
            "cont": cont,
            "sys_name": sys_name,
            "snap": snap_name,
            "epc": epc,
            "epcrange": epcrange,
        }
        return self._get_result(("container", "destroy-snap"), **kwargs)
```
Plus remove lines 427-429.

Reply:
Done